product-service/handlers/getProductsById.js

```diff
@@ -0,0 +1,21 @@
+import productList from './productList.json';
+import { headers } from '../constants/headers';
+
+export const getProductsById = async event => {
+  const { productId } = event.pathParameters;
+  try {
+    if (productId > productList.length - 1) throw new Error();
+
+    return {
+        statusCode: 200,
+        headers: headers,
+        body: JSON.stringify(productList[productId]),
```

Comment:
I found it confusing, that I can't search products by their actual Ids, like `7567ec4b-b10c-45c5-9345-fc73c48a80a1`. Could you please fix it to something like `productList.find(product => product.id === id)`. And throw error, if a product is not found.

Reply:
changed it

Reply:
and added async/await usage

Reply:
acknowledged + approved